lib/matplotlib/tests/test_axes.py

```diff
@@ -34,6 +34,7 @@
     assert_allclose, assert_array_equal, assert_array_almost_equal)
 from matplotlib import rc_context
 from matplotlib.cbook import MatplotlibDeprecationWarning
+from mpl_toolkits.axes_grid1 import make_axes_locatable
```

Comment:
this cannot go in the main library.  Please test in `mpl_toolkits/tests/`  Thanks!